config/config.go

```diff
@@ -244,6 +252,16 @@ type LoadBalancer struct {
 	SecurityGroupIds []string `yaml:"securityGroupIds"`
 }
 
+type Plugins struct {
+	Rbac map[string]string `yaml:"rbac"`
```

Comment:
Yes, making it an another struct would be better 👍
@gianrubio, did you have any chance to address it?